translations_to_weblate.sh

```diff
@@ -1,23 +1,15 @@
 #!/bin/bash
-target='*.tsl'
-output='BE_weblate.json'
+MY_PATH="`dirname \"$0\"`"
+TARGET='*.tsl'
+OUTPUT='BE_weblate.json'
 if [[ ! -z $1 ]] ; then
-    target=$1
+    TARGET=$1
 fi
 if [[ ! -z $2 ]] ; then
-    output=$2
+    OUTPUT=$2
 fi
-
-echo "CHANGING FORMAT TO WEBLATE"
-cat "${target}" > weblate_translations.tmp
-awk '
-        BEGIN { print "{"; }
-        {   if (NR==TNR)
-                {print "\t\""$0"\": \""$0"\"";}
-              else
-                  {print "\t\""$0"\": \""$0"\",";}
-        }
-        END { print "}" }
-    ' weblate_translations.tmp >>"${output}"
+echo "=== CHANGING FORMAT TO WEBLATE (json) ==="
+cat ${TARGET} > weblate_translations.tmp
```

Comment:
...and in the later usage of such values...

it may be not an issue in `bash`, but if the script gets evaluated by something else, it can backfire.

Reply:
this is actually intended, if you check for default target, it's *.tsl", so it wouldn't get evaluated.